dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts

```diff
@@ -20,7 +20,7 @@ export type OpenAPIConfig = {
 };
 
 export const OpenAPI: OpenAPIConfig = {
-    BASE: process.env.NODE_ENV === 'production' ? '/api' : 'http://127.0.0.1:8000',
+    BASE: 'http://localhost:8000', // process.env.NODE_ENV === 'production' ? '/api' : 'http://127.0.0.1:8000',
```

Comment:
I think this should stay as is

Reply:
This should be solved by https://github.com/brouberol/5esheets/issues/146. Should we keep it as-is in the meantime?

Reply:
Fixed by https://github.com/brouberol/5esheets/commit/dbe59856d2647e670d7c0acf224a68a037830a7a#diff-61178c0abe6fbdba6e7be2fcbc02e55ffb0939f433470a284f735730a8716da7R12-R14